language/move-model/src/builder/module_builder.rs

```diff
@@ -1,11 +1,13 @@
 // Copyright (c) The Diem Core Contributors
 // SPDX-License-Identifier: Apache-2.0
 
-use std::collections::{BTreeMap, BTreeSet};
+use std::{
+    collections::{BTreeMap, BTreeSet},
+    fmt,
+};
 
 use itertools::Itertools;
 use regex::Regex;
-use serde::export::{fmt, Formatter};
```

Comment:
I think it's funny because my IDE seems to default to this (probably why there's a bunch of these)

Reply:
That needs to get reported as a bug to whatever IDE or language plugin you are using. It should not be inserting imports of hidden paths.